sashimi-plot.py

```diff
@@ -278,15 +278,21 @@ def read_gtf(f, c):
         transcripts = OrderedDict()
         chr, start, end = parse_coordinates(c)
         end = end -1
-        with open(f) as openf:
+        with gzip.open(f, "rt") if f.endswith(".gz") else open(f, "r") as openf:
                 for line in openf:
                         if line.startswith("#"):
                                 continue
                         el_chr, _, el, el_start, el_end, _, strand, _, tags = line.strip().split("\t")
                         if el_chr != chr:
                                 continue
+                        if el not in ("transcript", "exon"):
+                                continue
                         d = dict(kv.strip().split(" ") for kv in tags.strip(";").split("; "))
-                        transcript_id = d["transcript_id"]
+                        try:
+                                transcript_id = d["transcript_id"]
```

Comment:
```suggestion
                                transcript_id = re.findall('transcript_id ("[^"]+")', tags)[0]
```
As it affects the same portion of code I take advantage to suggest this line too, which fixes #48